src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetBuilding.java

```diff
@@ -109,15 +118,17 @@ public void addInformation(ItemStack stack, @Nullable World world, List<String>
     @Override
     public EnumActionResult onItemUse(EntityPlayer player, World world, BlockPos pos, EnumHand hand, EnumFacing side, float hitX, float hitY, float hitZ) {
         //On item use, if sneaking, select the block clicked on, else build -- This is called when a block in clicked on
+        if (!world.isRemote)
```

Comment:
```suggestion
        if (world.isRemote)
```

Reply:
Oh, I actually missed that... XD
Thought what I meant is that in the original implementation always returned SUCCESS, though now there is a case where it returns FAIL - but I guess that returning Failure is the way to go...